docs/pages/reference/access-controls/roles.mdx

```diff
@@ -142,6 +144,52 @@ spec:
       'workload': ['database', 'backup']
 ```
 
+### `client_idle_timeout`
+
+This example configures a global client idle timeout of **30 minutes**. After 30 minutes of inactivity, the session will be terminated.
+
+The `client_idle_timeout` in Teleport is a configurable setting that helps improve security by terminating inactive sessions after a specified period. It can be applied globally or per role, allowing for flexibility based on your organization's security policies.
```

Comment:
I would split this paragraph in two or condense it, since for me it's pretty long.